packages/odyssey-storybook/src/components/odyssey-mui/Form/Form.stories.tsx

```diff
@@ -265,6 +266,14 @@ export const Simple = Template.bind({});
 Simple.args = {
   fieldGroupOne: true,
 };
+Simple.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const dropdown = await canvas.getByLabelText("Docking license");
+  const textarea = await canvas.getByLabelText("Nature of visit");
+  await userEvent.click(textarea);
+  await userEvent.type(textarea, "business");
+  await userEvent.click(dropdown);
+};
```

Comment:
When someone's selecting stuff with a screenreader, I think they'll typically be finding things by role, not just label text.

So in this case, we'll want `getByRole('whatever', { name: "Docking license" })`.

The text area is `textbox`. And I think there's a property for `multiline`.

Reply:
Isn't it that you go through all fields in the form and choose from there? You'll be selecting a textbox role with a certain text.

Either way, this is non-blocking.